api/src/opentrons/protocols/labware.py

```diff
@@ -146,15 +166,24 @@ def verify_definition(
     if isinstance(contents, dict):
         to_return = contents
     else:
-        to_return = json.loads(contents)
+        try:
+            to_return = json.loads(contents)
+        except json.JSONDecodeError as e:
+            raise NotALabwareError("invalid-json", [e])
```

Comment:
Nitpick: `from e` so the tracebacks reflect that we're deliberately converting exception `A` to exception `B`, instead of encountering an unexpected exception `A` while we're trying to handle exception `B`.

```suggestion
            raise NotALabwareError("invalid-json", [e]) from e
```

`from None` instead of `from e` would also make sense here, just a matter of taste I think.

Reply:
oh whoops yeah good call